### PR TITLE
✨ feat: SQ-65941 [front] Criar componente do guia de apresentação - 1

### PR DESCRIPTION
**Pull request by wandersonsales-dev**
**Link da Tarefa:**

[SQ-65941](https://duopana.myjetbrains.com/youtrack/issue/SQ-65941) [front] Criar componente do guia de apresentação - 1

[SQ-65942](https://duopana.myjetbrains.com/youtrack/issue/SQ-65942) [front] Criar componentes do guia de apresentação -2

[SQ-65943](https://duopana.myjetbrains.com/youtrack/issue/SQ-65943) [front] Criar componentes do guia de apresentação - 3

- [x] O PR está com o nome no formato **(SQ-12345) Título da Tarefa**
- [x] Foi completamente testado
- [x] Merge do branch **master** foi feito antes do PR ser criado

## Documentação:

- [ ] Criou novos componentes? foi criado a documentação (.md)
- [ ] Criou testes para os novos componentes

## Evidências de Teste:

https://www.awesomescreenshot.com/video/31866218?key=fea4469965fa31a9fd58d47d0105724c

- [x] Possui evidências na tarefa

## Release Notes

- **New Features**
  - Introduced a new modal component for welcoming creators with insights, featuring navigation and translation support.
  - Added a Storybook entry for the new modal component to showcase its various states.
  - Implemented swipe functionality for mobile users to navigate through modal pages.

- **Localization**
  - Added localization resources for the new modal in English, Portuguese, and Spanish.

- **Style**
  - Updated styles for modal components to improve visual design and layout.

### CODE REVIEW

File: src/components/sq-modal/sq-modal.component.scss [hunk 1]
```diff
@@ -1,16 +1,16 @@
...
-  }
-  
\ No newline at end of file
+}
+.modal-over {
+  padding: 0;
+}
+.force-mobile-no-margin {
+  margin: 0 auto !important;
+}
```
[the remark this answers is not in the document]
Comment (wandersonsales-dev) [reply src/components/sq-modal/sq-modal.component.scss:16]: It's not necessary

File: src/components/sq-modal/sq-modal.component.scss [hunk 2]
```diff
@@ -1,16 +1,16 @@
...
+  .modal-exit {
+    margin: 0;
     padding: 0;
   }
-  .force-mobile-no-margin {
-    margin: 0 auto !important;
-  }
-  
\ No newline at end of file
+}
```
[the remark this answers is not in the document]
Comment (wandersonsales-dev) [reply src/components/sq-modal/sq-modal.component.scss:10]: It's not necessary

File: src/components/sq-modal-welcome-creators-insights/__docs__/sq-modal-steps.component.stories.tsx [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+    // pages: [
+    //   <div key="1">
+    //     <button>prev</button>
+    //     <button>next</button>
+    //   </div>,
+    //   <div key="2">Page 2</div>,
+    //   <div key="3">Page 3</div>,
+    // ],
+  },
+}
```
[the remark this answers is not in the document]
Comment (wandersonsales-dev) [reply src/components/sq-modal-welcome-creators-insights/__docs__/sq-modal-steps.component.stories.tsx:28]: Done :rocket:

File: src/components/sq-flow/sq-flow.component.tsx [hunk 4]
```diff
@@ -12,19 +14,21 @@ export interface Props extends React.HTMLAttributes<HTMLDivElement> {
...
+        {steps?.length
+          ? Array.from(Array(steps?.length))?.map((step, index) => {
+              return (
+                <li
+                  key={index}
+                  className={`${index === active ? 'active' : ''}`}
+                  style={{ backgroundColor: active === index ? activeColor : '' }}
+                  data-tooltip-id="tooltip"
+                  data-tooltip-content={step?.status}
+                ></li>
```
[the remark this answers is not in the document]
Comment (wandersonsales-dev) [reply src/components/sq-flow/sq-flow.component.tsx:28]: It's not necessary

File: src/components/sq-flow/sq-flow.component.tsx [hunk 5]
```diff
@@ -12,19 +14,21 @@ export interface Props extends React.HTMLAttributes<HTMLDivElement> {
...
+              )
+            })
+          : null}
```
[the remark this answers is not in the document]
Comment (wandersonsales-dev) [reply src/components/sq-flow/sq-flow.component.tsx:31]: It's not necessary

File: src/components/sq-modal-welcome-creators-insights/sq-modal-welcome-creators-insights.component.scoped.scss [hunk 6]
```diff
@@ -0,0 +1,50 @@
...
+
+      span {
+        i {
+          font-size: 8px;
+          position: relative;
+          top: 1px;
+          color: var(--primary_color);
+        }
+      }
+    }
```
[the remark this answers is not in the document]
Comment (wandersonsales-dev) [reply src/components/sq-modal-welcome-creators-insights/sq-modal-welcome-creators-insights.component.scoped.scss:21]: This value should be fixed

File: src/components/sq-modal-welcome-creators-insights/sq-modal-welcome-creators-insights.component.scoped.scss [hunk 7]
```diff
@@ -0,0 +1,50 @@
...
+        width: 80%;
+      }
+
+      img {
+        width: 90%;
+        height: auto;
+        object-fit: contain;
+        z-index: 1;
+      }
+    }
```
[the remark this answers is not in the document]
Comment (wandersonsales-dev) [reply src/components/sq-modal-welcome-creators-insights/sq-modal-welcome-creators-insights.component.scoped.scss:38]: It's not necessary

File: src/components/sq-modal-welcome-creators-insights/sq-modal-welcome-creators-insights.component.scoped.scss [hunk 8]
```diff
@@ -0,0 +1,50 @@
...
+    .footer {
+      width: calc(100% + 2rem);
+      left: -1rem;
+      background-color: var(--background_third);
+      height: auto;
+      margin-top: -33px;
+      flex-direction: column;
+      gap: 1rem;
+    }
+  }
```
[the remark this answers is not in the document]
Comment (wandersonsales-dev) [reply src/components/sq-modal-welcome-creators-insights/sq-modal-welcome-creators-insights.component.scoped.scss:49]: This causes a different behavior than desired. When using top the space previously occupied at the end of the div will still be considered, while with margin-top this does not happen.

File: src/components/sq-modal-welcome-creators-insights/__docs__/sq-modal-steps.component.example.tsx [hunk 9]
```diff
@@ -0,0 +1,35 @@
...
+      <SqModalWelcomeCreatorsInsights
+        {...props}
+        open={open}
+        onCloseChange={() => handleClose()}
+        onOpenChange={(open) => setOpen(open)}
+        buttonClose={false}
+        forceMobileNoMargin
+      />
+    </div>
+  )
```
[the remark this answers is not in the document]
Comment (wandersonsales-dev) [reply src/components/sq-modal-welcome-creators-insights/__docs__/sq-modal-steps.component.example.tsx:32]: Done :rocket:

**Review by elianefaveron1 — APPROVED** [review]

**Comment by wandersonsales-dev** [comment]
@JonasPeres code review

**Review by htamagnus — DISMISSED** [review]

**Review by JonasPeres — DISMISSED** [review]

File: src/components-creators-insights/sq-modal-welcome-creators-insights/__docs__/sq-modal-welcome-creators-insights.component.stories.tsx [hunk 10]
```diff
@@ -0,0 +1,16 @@
...
+  component: SqModalWelcomeCreatorsInsightsExample,
+  tags: ['autodocs'],
+}
+
+export default meta
+type Story = StoryObj<typeof SqModalWelcomeCreatorsInsightsExample>
+
+export const Default: Story = {
+  args: {},
+}
```
[the remark this answers is not in the document]
Comment (wandersonsales-dev) [reply src/components-creators-insights/sq-modal-welcome-creators-insights/__docs__/sq-modal-welcome-creators-insights.component.stories.tsx:16]: It's not necessary

File: src/components-creators-insights/sq-modal-welcome-creators-insights/sq-modal-welcome-creators-insights.component.scoped.scss [hunk 11]
```diff
@@ -0,0 +1,50 @@
+.sq-modal-steps {
+  .content {
+    width: calc(100% + 2rem);
+    left: -1rem;
+    flex-direction: column;
```
[the remark this answers is not in the document]
Comment (wandersonsales-dev) [reply src/components-creators-insights/sq-modal-welcome-creators-insights/sq-modal-welcome-creators-insights.component.scoped.scss:5]: It's wrong

File: src/components-creators-insights/sq-modal-welcome-creators-insights/sq-modal-welcome-creators-insights.component.scoped.scss [hunk 12]
```diff
@@ -0,0 +1,50 @@
...
+    .footer {
+      width: calc(100% + 2rem);
+      left: -1rem;
+      background-color: var(--background_secondary);
+      height: auto;
+      margin-top: -33px;
+      flex-direction: column;
+      gap: 1rem;
+    }
+  }
```
[the remark this answers is not in the document]
Comment (wandersonsales-dev) [reply src/components-creators-insights/sq-modal-welcome-creators-insights/sq-modal-welcome-creators-insights.component.scoped.scss:49]: It's wrong

File: src/components-creators-insights/sq-modal-welcome-creators-insights/__docs__/sq-modal-welcome-creators-insights.component.example.tsx [hunk 13]
```diff
@@ -0,0 +1,35 @@
...
+  return (
+    <div
+      style={{
+        display: 'flex',
+        justifyContent: 'center',
+        alignItems: 'center',
+        height: '100%',
+        minHeight: '300px',
+      }}
+    >
```
[the remark this answers is not in the document]
Comment (wandersonsales-dev) [reply src/components-creators-insights/sq-modal-welcome-creators-insights/__docs__/sq-modal-welcome-creators-insights.component.example.tsx:21]: It's not necessary

File: src/components-creators-insights/sq-modal-welcome-creators-insights/__docs__/sq-modal-welcome-creators-insights.component.example.tsx [hunk 14]
```diff
@@ -0,0 +1,35 @@
...
+      <SqButtonComponent onClick={() => setOpen(true)}>Open Modal</SqButtonComponent>
+      <SqModalWelcomeCreatorsInsights
+        {...props}
+        open={open}
+        onCloseChange={() => handleClose()}
+        onOpenChange={(open) => setOpen(open)}
+        buttonClose={false}
+        forceMobileNoMargin
+      />
+    </div>
```
[the remark this answers is not in the document]
Comment (wandersonsales-dev) [reply src/components-creators-insights/sq-modal-welcome-creators-insights/__docs__/sq-modal-welcome-creators-insights.component.example.tsx:31]: It's not necessary

**Review by htamagnus — DISMISSED** [review]

**Comment by wandersonsales-dev** [comment]
@JonasPeres code review

**Review by JonasPeres — APPROVED** [review]

**Review by JonasPeres — APPROVED** [review]

**Review by htamagnus — DISMISSED** [review]

**Review by elianefaveron1 — APPROVED** [review]

**Review by elianefaveron1 — CHANGES_REQUESTED** [review]
Tem conflitos

**Review by htamagnus — COMMENTED** [review]
só precisa resolver o conflito

**Review by htamagnus — APPROVED** [review]

**Review by JonasPeres — APPROVED** [review]

**Review by elianefaveron1 — APPROVED** [review]

**Pull request merged**